Developers/Writing_Good_Commits.rst

```diff
@@ -7,9 +7,8 @@ At coala we are looking heavily at the maintainability of the code.
 
     Code is more often read than written!
 
-It is obvious that we need good code. In order to do that we are
-verifying that every change to our code (i.e. the commits) is making it
-better.
+It is obvious that we need good code. Therefore we are ensuring that 
```

Comment:
Again, whitespaces. It's in this line, something line `<writed-line> ` . That ` ` is a whitespace